### PR TITLE
LPS-147531 The business type label should be displayed when adding field in the layout

### PR DESCRIPTION
**Pull request by marcelabc**
Previous Review: https://github.com/liferay-objects/liferay-portal/pull/432

Related issue: https://issues.liferay.com/browse/LPS-147531

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
I'm going to fix the rebase error.

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:reevaluate:1367401_547

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:reevaluate:1317401_2337

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/util/ObjectFieldBusinessTypeUtil.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * @author Marcela Cunha
+ */
+public class ObjectFieldBusinessTypeUtil {
+
+	public static List<Map<String, String>> getObjectFieldBusinessTypeMaps(
+		List<ObjectFieldBusinessType> objectFieldBusinessTypes, Locale locale) {
```
Comment (gabrielwas) [inline modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/util/ObjectFieldBusinessTypeUtil.java:31]: @marcelabc can you sort here?

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
I'm changing the labels because I fixed the minor issue.

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**